bundle/language_en.properties

```diff
@@ -891,6 +891,7 @@ angal.menu.btn.examtype
 angal.menu.btn.generaldata                                                                             = Settings
 angal.menu.btn.groups                                                                                  = Groups
 angal.menu.btn.help                                                                                    = Help
+angal.menu.btn.logout																				   = Logout
```

Comment:
fix alignment please

Reply:
I don't see a wrong alignment

Reply:
not?
![image](https://user-images.githubusercontent.com/2938553/202064188-74096469-b510-4022-9e97-faa1fa0bb9e4.png)

Reply:
I don't see it on linux. I replaced tabs with spaces.